heliopy/data/util.py

```diff
@@ -189,6 +189,7 @@ def units_attach(data, units):
     out : sunpy.timeseries.timeseriesbase.GenericTimeSeries
         DataFrame converted into TimeSeries with units attached.
     """
+    warnings.warn("Returning units", RuntimeWarning)
```

Comment:
What is this warning for?

Reply:
It's just a temporary warning to see what functions are currently using the units_attach method. I will remove it later on

Reply:
Removing it now since your way sounds much more better!